website/src/routes/guides/(get-started)/ecosystem/index.mdx

```diff
@@ -41,4 +41,4 @@ Use the button at the bottom left of this page to add your project to this ecosy
 
 ### Utilities
 
-- No entry yet - be quick to be the first
+- [@nest-lab/typeschema](https://github.com/jmcdo29/nest-lab/tree/main/packages/typeschema): A ValidationPipe that handles many schema validators in a class-based fashion for NestJS's input validation
```

Comment:
Ist "ValidationPipe" a typo?

Reply:
It's a term from NestJS. If you'd prefer I can separate it out though

Reply:
No, that's fine the way it is.